src/main/java/UI/searchPageHelpFrame.java

```diff
@@ -64,15 +60,15 @@ public searchPageHelpFrame(UserEnterIngredientsController ingredientsController,
         panel1.setLayout(null);
 
         //create open_fridge button which pops up a messagebox of what ingredients fridge has.
-        open_fridge = new JButton("open your fridge");
-        open_fridge.setBounds(300, 80, 240, 80);
-        open_fridge.addActionListener(this);
+//        open_fridge = new JButton("open your fridge");
+//        open_fridge.setBounds(300, 80, 240, 80);
+//        open_fridge.addActionListener(this);
```

Comment:
Delete the open_fridge code we will no longer use it.

Reply:
Deleted